index.html

```diff
@@ -38,5 +38,26 @@ <h2 class="text-container" id="sdsection">
       <img src="arrow-button.svg" class="arrow-button"><a href="#"></a></img>
       </div>
     </section>
+    <section class="works">
+      <h1 class="recent-works">My Recent Works</h1>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div>
+    </section>
+    <section class="about-me">
+      <h2 class="about-me-title">About me</h2>
+      <p class="about-me-description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </section>    
```

Comment:
- [ ] Your about me section is missing some background images, it doesn't look like what is in the Figma design. Please refer to the image below and fix this as well :100: 
- Your design
![image](https://user-images.githubusercontent.com/102699958/212463490-abc46921-3939-483c-a56d-1703ca4f6c6f.png)
- The  Figma Design
![image](https://user-images.githubusercontent.com/102699958/212463513-d3480c45-31d5-4b3c-95ca-71decec2ea82.png)

- [ ] Your about me section is also incomplete. It is missing the technology categories ("Languages", "Frameworks" and "Skills"). Please add these to your project as well and kindly ensure that they conform to the project requirements and look like the Figma design :+1: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/212463683-c893e630-036c-42fa-b0e0-c76ab4e96dff.png)